lib/utils.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+function getReportNode(argNode, node) {
+    return argNode || node.callee;
+}
+
+function isStringLiteral(node) {
+    return node && node.type === 'Literal' && typeof node.value === 'string';
+}
+
+const hasMultipleVaribles = str => {
+    if (str.match(/%[sd]/gm) && str.match(/%\d+\$(s|d)/gm)) {
```

Comment:
Why are we doing the first check here?

We can also simplify the second regex to `/%\d+\$[sd]/gm`

Reply:
Fixed👍